feature-changeaddress/src/main/kotlin/com/hedvig/android/feature/changeaddress/ui/QuoteCard.kt

```diff
@@ -0,0 +1,210 @@
+import com.hedvig.android.core.ui.UiMoney
+import com.hedvig.android.feature.changeaddress.data.Address
+import com.hedvig.android.feature.changeaddress.data.AddressId
+import com.hedvig.android.feature.changeaddress.data.MoveIntentId
+import com.hedvig.android.feature.changeaddress.data.MoveQuote
+import kotlinx.datetime.LocalDate
+import octopus.type.CurrencyCode
+
+@Composable
+fun QuoteCard(
+  movingDate: String?,
+  quote: MoveQuote,
+  onExpandClicked: () -> Unit,
+  isExpanded: Boolean,
+) {
+  Column(
+    modifier = Modifier
+      .padding(horizontal = 16.dp)
+      .clip(SquircleShape)
+      .clickable { onExpandClicked() }
+      .background(shape = SquircleShape, color = Color(0xFFF0F0F0))
+      .padding(16.dp),
+  ) {
+    Row(verticalAlignment = Alignment.CenterVertically) {
+      Image(
+        painter = painterResource(id = R.drawable.ic_pillow),
+        contentDescription = "",
+        modifier = Modifier.size(48.dp),
+      )
+      Spacer(modifier = Modifier.padding(start = 16.dp))
+      Column {
+        Text(
+          text = "Hemförsäkring Bostadsrätt",
+          style = MaterialTheme.typography.titleMedium,
+          fontSize = 18.sp,
+        )
+        Row(verticalAlignment = Alignment.CenterVertically) {
+          Text(
+            text = "Aktiveras $movingDate",
+            style = MaterialTheme.typography.titleMedium,
+            color = Color(0xFF727272),
+            fontSize = 18.sp,
+          )
+          Spacer(modifier = Modifier.padding(start = 5.dp))
+          Icon(
+            painter = painterResource(id = com.hedvig.android.core.designsystem.R.drawable.ic_info),
+            tint = Color(0xFF727272),
+            contentDescription = "",
+          )
+        }
+      }
+    }
+
+    Spacer(modifier = Modifier.padding(top = 16.dp))
+
+    Divider(
+      thickness = 1.dp,
+      color = Color(0xFFCFCFCF),
+    )
+
+    Spacer(modifier = Modifier.padding(top = 8.dp))
+
+    Row(
+      modifier = Modifier.fillMaxWidth(),
+      horizontalArrangement = Arrangement.Absolute.SpaceBetween,
+      verticalAlignment = Alignment.CenterVertically,
+    ) {
+      Row(
+        verticalAlignment = Alignment.CenterVertically,
+        modifier = Modifier.padding(vertical = 8.dp),
+      ) {
+        Text(
+          text = "Detaljer",
+          style = MaterialTheme.typography.titleMedium,
+          fontSize = 18.sp,
+        )
+        Spacer(modifier = Modifier.padding(4.dp))
+
+        if (isExpanded) {
+          Icon(
+            painter = painterResource(id = com.hedvig.android.core.designsystem.R.drawable.ic_drop_down_indicator_expanded),
+            contentDescription = "",
+            tint = Color(0xFFB4B4B4),
+          )
+        } else {
+          Icon(
+            painter = painterResource(id = com.hedvig.android.core.designsystem.R.drawable.ic_drop_down_indicator),
+            contentDescription = "",
+            tint = Color(0xFFB4B4B4),
+          )
+        }
+      }
```

Comment:
If you're feeling like adding more animation to this, you can do something like this
```suggestion
var angle = animateFloatAsState(
    targetValue = if (isExpanded) { 0f } else { 180f },
    // animationSpec = spring(), // Might wanna play with the animation spec if the default spring feels odd
  )
  Icon(
    painter = painterResource(id = com.hedvig.android.core.designsystem.R.drawable.ic_drop_down_indicator),
    contentDescription = "",
    tint = Color(0xFFB4B4B4),
    modifier = Modifier.graphicsLayer {
      rotationZ = angle
    }
  )
```
So that you don't need to add the new icon resource at all, and you get the original icon to rotate clockwise to the new position.
If it rotates on the wrong side, play around with the initial and target values, you may wanna start at 180 degrees and go to 360 or something like that, not sure, I didn't run this but it should work.